### ISSUE TITLE
Makefile develop target broken

### ISSUE THREAD

**Issue by james-mchugh** [faust-streaming/faust#132]
## Checklist

- [x] I have included information about relevant versions
- [x] I have verified that the issue persists when using the `master` branch of Faust.

## Steps to reproduce

Run `make develop`

## Expected behavior

`make develop` runs properly 

## Actual behavior

Running `make develop` fails because default.txt is not in requirements directory.

## Full traceback

```pytb
(faust) ➜  faust git:(master) make develop
pip install -U -r requirements/default.txt
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements/default.txt'
make: *** [reqs-default] Error 1
```

# Versions

* Python version 3.9.4
* Faust version master
* Operating system Darwin 19.6.0
* Kafka version na
* RocksDB version (if applicable)

**Comment by taybin** [comment]
Good catch.  I fixed this, but then I saw that `make develop` still breaks because it runs `pip install -r requirements/dist.txt`, and that references a non-existent `flakes.txt` file.  Should the flake8 linters be moved to that?  Or should the line be deleted from `dist.txt`?  Or is the `Makefile` not really used anymore?  In that case, maybe it should be trimmed down to what is used or even deleted entirely.
